### DISCUSSION TITLE
Intelligent Octopus Go and Ohme integration

### DISCUSSION THREAD

**Discussion by sdss1234** [BottlecapDave/HomeAssistant-OctopusEnergy#1048] [Q&A]
Thanks in advance for any help that anyone can offer - 

I've seen the discussion about Ohme charger here - https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/573 - and it looks like the Ohme charger works in a curious way compared to the majority of chargers on the Intelligent Octopus Go tarriff.

Does anyone know how to reliably get the integration reporting the smart charging times?  Or is it entirely based on the Ohme app? (i'll need to use the times to stop solar batteries being used to charge the car etc)

thanks

**Comment by BottlecapDave** [comment]
Hello. Unfortunately the data available via OE APIs is not sufficient to determine charging times for Ohme users, which is why it's not exposed via this integration. There is [this HA integration](https://github.com/dan-r/HomeAssistant-Ohme) but I'm not sure if it exposes the data you require.

**Reply by bruceskingle** [reply]
Get an electrician to move the ev charger outside the ct clamp for your battery, then it will never see the ev charge load.

https://youtu.be/dMiuiFQnyPA?si=zG4DES8WaNMZitMu
